docs/conf.py

```diff
@@ -41,6 +41,9 @@
 
 project = 'PyERFA'
 author = 'The PyERFA Developers'
+build_date = datetime.utcfromtimestamp(
+    int(os.environ.get('SOURCE_DATE_EPOCH', time.time()))
+)
 copyright = '2011–{0}, {1}'.format(datetime.utcnow().year, author)
```

Comment:
shouldnt this use `build_date.year` here?

Reply:
thanks @bmwiedemann you are perfectly right.
I will fix it ASAP

Reply:
[The original patch from Debian is unaffected](https://bugs.debian.org/cgi-bin/bugreport.cgi?att=1;bug=964958;filename=pyerfa.diff.txt;msg=5) - it was the version here that was copied incorrectly.